qgis-project/qwat.qgs

```diff
@@ -88941,18 +88944,7 @@ def my_form_open(dialog, layer, feature):
         </spatialrefsys>
       </srs>
       <provider encoding="UTF-8">postgres</provider>
-      <previewExpression>coalesce(identification|| ' ','') || CASE
-WHEN  "installation_type"  = 'chamber' THEN 'C. '
-WHEN "installation_type"  = 'pressurecontrol' AND fk_pressurecontrol_type = 2801 THEN 'CR. '
-WHEN  "installation_type"  = 'pressurecontrol' AND fk_pressurecontrol_type = 2802 THEN 'CCP. '
-WHEN  "installation_type"  = 'pressurecontrol' AND fk_pressurecontrol_type = 2803 THEN 'CRA. '
-WHEN  "installation_type"  = 'pump' THEN 'P. '
-WHEN  "installation_type"  = 'source' THEN 'S. '
-WHEN  "installation_type"  = 'tank' THEN 'R. '
-WHEN  "installation_type"  = 'treatment' THEN 'T. '
-ELSE ''
-END
-||name</previewExpression>
```

Comment:
I haven't removed the expression from the label, but from the relation widget on fk_installation so that it shows the installation id and not what what's in ('CR.','CCP','CRA' etc.).

So, what's presented here:
![cover](https://cloud.githubusercontent.com/assets/3179646/13568981/e360f2b2-e46d-11e5-8f74-a73cea86d9c9.png)

Are you ok with it or do you want me to have in the widget the same expression in the label?